app/scripts/lforms-controllers.js

```diff
@@ -950,7 +962,6 @@ angular.module('lformsWidget')
 
           // for debug only. to be removed.
         $scope.onclick = function() {
-          debugger
```

Comment:
Is just this single statement "for debug", is should the whole block be removed.

Actually, I don't see the reason to delete this in our fork. It will only go into the debugger when you have the developer console open, right?